packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -221,7 +221,7 @@ function FacetFilterControls(props: FacetFilterControlsProps) {
 
   return (
     <div className={`FacetFilterControls`}>
-      {combineRangeFacetConfig && (
+      {combineRangeFacetConfig && combinedRangeFacets.length >= 2 && (
```

Comment:
Fix bug where `combinedRangeFacets` was defined but did not have enough elements.